### PR TITLE
Feat/create events model

### PR DESCRIPTION
**Pull request by raissaheimann**
## What I did

- Create events.js on models 
- Create all the columns about events:
1. name
2. date
3. images
4. description
5. link
6. simpleDescription

## How to test
1. Open the terminal and go to the folder of this project
2. In terminal type:
3. git pull
4. git checkout feat/create-events-model
5. cd server/
6. node install
7. node index.js
8. In your browser open the http://localhost:3001/admin
9. Open events and create a event
10. Put a name and hold create
11. Put the other informations(date, images, description, link and simple description)
12. Press create and that's it :)

### CODE REVIEW

File: server/models/Post.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ Post.add({
     index: true,
   },
   image: {
-    type: Types.CloudinaryImages,
+    type: Types.CloudinaryImage,
```
Comment (viniciusscala) [inline server/models/Post.js]: This will probably break the front end

**Review by viniciusscala — CHANGES_REQUESTED** [review]

File: server/models/Post.js (hunk 1 above)
Comment (guilhermefxs) [inline server/models/Post.js]: This will probably break the front end, don't take the 's' off of Types.CloudinaryImages

**Review by viniciusscala — CHANGES_REQUESTED** [review]
The name of the models files should start with a uppercase latter

**Review by viniciusscala — APPROVED** [review]
Perfect 😁

**Review by Monterazo — CHANGES_REQUESTED** [review]
Command node index.js doesn't work and sends the following error:

Error: Invalid Configuration

Field (Events.images) is required but not initial, and has no default or generat
ed value.
Please provide a default, remove the required setting, or set initial: false to
override this error.

**Review by Monterazo — APPROVED** [review]
It seems ok!

**Review by guilhermefxs — APPROVED** [review]

**Pull request merged**